backtesting/backtesting.py

```diff
@@ -1163,6 +1174,8 @@ def run(self, **kwargs) -> pd.Series:
 
     def optimize(self,
                  maximize: Union[str, Callable[[pd.Series], float]] = 'SQN',
+                 method: Literal['auto', 'full', 'random', 'skopt'] = 'auto',
```

Comment:
The reason [CI breaks](https://travis-ci.org/github/kernc/backtesting.py/builds/736952138) is that [`Literal` was only included in Python 3.8](url), but we support 3.5+. You'll have to use a [type comment](https://www.python.org/dev/peps/pep-0484/#type-comments), or untype the line completely. :confused: